alternatives.c

```diff
@@ -1300,7 +1300,7 @@ int main(int argc, const char **argv) {
     struct alternative newAlt = {-1, {NULL, NULL, NULL}, NULL, NULL, 0, NULL};
     int flags = 0;
     char *altDir = "/etc/alternatives";
-    char *stateDir = "/var/lib/alternatives";
+    char *stateDir = "/etc/alternatives/state";
```

Comment:
I think @travier is right, this could in theory conflict if someone introduced an alternative called `state`, so we need an alternative alternatives dir here. Possibly `/etc/alternatives.state` or `/etc/.alternatives.state` so?